### DISCUSSION TITLE
backup x-ui

### DISCUSSION THREAD

**Discussion by AlefbeMedia** [FranzKafkaYu/x-ui#140] [Q&A] [answered]
hi.
By chance, my server had a problem and x-ray stopped working, and no matter what I do, it doesn't work.
I have many users on the server.
How can I make a backup of the panel and user information and then reset the server?

**Comment by FranzKafkaYu** [comment, marked as answer]
@AlefbeMedia just export /etc/x-ui/x-ui.db to  a new server

**Reply by sina-p** [reply]
This method only works for the same version of X-UI. If I want to switch between the old X-UI and this version, the database file does not work and X-ray gives an error and all my users are lost.
plz I need your help🙏🏻🙏🏻
